apps/api/docs/channels/mailgun.md

```diff
@@ -4,26 +4,26 @@ Mailgun is an email service provider that offers a RESTful API for sending email
 
 ### Values to Update in Database
 
-When using Mailgun to send emails via their API, you need to provide certain variables that hold the Mailgun configuration details. Here are the values you need to update in table `notify_providers`:
+When using Mailgun to send emails via their API, you need to provide certain variables that hold the Mailgun configuration details. Here are the values you need to update in the table `notify_providers`:
 
-Create a new entry in table `notify_providers` and set the fields - `name`, `application_id`, `user_id`
+Create a new entry in the table `notify_providers` and set the fields - `name`, `application_id`, `user_id`.
 
 - Set field `channel_type` = 2 (for Mailgun)
 - Set field `is_enabled` = 1 (to enable the newly created provider)
 
-Then set the following configurations in the `configuration` field
+Then set the following configurations in the `configuration` field:
 
 | Key             | Description                                                     |
 |-----------------|-----------------------------------------------------------------|
 | MAILGUN_API_KEY | Your Mailgun API key                                            |
 | MAILGUN_HOST    | Mailgun host; api.mailgun.net for US, api.eu.mailgun.net for EU |
 | MAILGUN_DOMAIN  | Your Mailgun domain name                                        |
 
-```jsonc
-// Sample json to set in configuration field
+```json
```

Comment:
Can we continue to use `jsonc` here as without it, comments get highlighted in red as invalid code in readme:
![image](https://github.com/OsmosysSoftware/osmo-x/assets/108812833/2a220d1c-c2dc-4f52-8da8-4c2db9cc5364)

Reply:
Reverted Back